### PR TITLE
add port to podman remote command

### PR DESCRIPTION
**Pull request by baude**
Signed-off-by: Brent Baude <bbaude@redhat.com>

### CODE REVIEW

File: pkg/domain/infra/tunnel/containers.go [hunk 1]
```diff
@@ -381,7 +381,29 @@ func (ic *ContainerEngine) Config(_ context.Context) (*config.Config, error) {
...
+		namesOrIds = append(namesOrIds, nameOrId)
+	}
+	ctrs, err := getContainersByContext(ic.ClientCxt, options.All, namesOrIds)
+	if err != nil {
+		return nil, err
+	}
+	for _, con := range ctrs {
+		if con.State != define.ContainerStateRunning.String() {
+			continue
+		}
```
Comment (TomSweeneyRedHat) [inline pkg/domain/infra/tunnel/containers.go:398]: Not sure, but I don't think we should add them to paused containers either.

Comment (rhatdan) [reply pkg/domain/infra/tunnel/containers.go:398]: This code is ignoreing all non running containers including paused containers, I believe?

**Comment by rhatdan** [comment]
LGTM

File: pkg/domain/infra/tunnel/containers.go (hunk 1 above)
Comment (baude) [reply pkg/domain/infra/tunnel/containers.go:398]: should be

Comment (TomSweeneyRedHat) [reply pkg/domain/infra/tunnel/containers.go:398]: DOH, don't know what I was reading earlier, ignore that comment.

**Pull request merged**